examples/compiler/03_iet.ipynb

```diff
@@ -0,0 +1,430 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part III - Halo region"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will know how is the `Halo` region being considered for building expressions up. For that, we will use the time marching example shown in `01_iet` tutorial. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from devito import Eq, Grid, TimeFunction, Operator\n",
+    "\n",
+    "grid = Grid(shape=(3, 3))\n",
+    "u = TimeFunction(name='u', grid=grid)\n",
+    "u.data[:] = 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At this moment, we have a time-varying $3x3$ grid filled with $1's$. As we can see below:"
```

Comment:
don't use DOLLARS ($) to highlight content, unless it's math (in fact, use google to find out what $ are recommended for)
Instead, you can use _ for italics, * for bold, etc. Up to you, but try to make it as latek-ish (and as nice) as possible

this applies to the whole document